### PR TITLE
V18.0.4 p2c

### PR DESCRIPTION
**Pull request by datalogics-joffref**
Release_Notes.html v18.0.4P2c

### CODE REVIEW

**Review by datalogics-pgallot — APPROVED** [review]
The information seems correct for 18.0.4+P2c, and the formatting seems to match the formatting used for previous entries.

While outside of the scope of the changes for 18.0.4+P2c, it does not seem like the article element is being properly used. Per https://www.w3.org/TR/2012/WD-html-markup-20121025/article.html#article the article element must have both a start tag and an end tag.
In our context, the article element should possibly be used to enclose each releases' release notes.

File: apdfl18/Release_Notes.html [hunk 1]
```diff
@@ -23,7 +23,26 @@ <h1>Release Notes, Adobe PDF Library v18</h1>
 <article id="page-8849" class="post-8849 page type-page status-publish hentry category-adobe-pdf-library">
    
 	<div class="entry-content"><p>This page lists updates, fixes, changes and enhancements made to the Adobe PDF Library v18 provided by Datalogics.</p>
-<p><strong>APDFL v18.0.4PlusP2b</strong> (July 31, 2023)</p>
+
+	<p><strong>APDFL v18.0.4PlusP2c</strong> (September 5, 2023)</p>
+<p>New Features:</p>
+<ul type="disc">
+<li>Creates new package Adobe.PDF.Library.Resource containing the APDFL Runtime Resources.</li>
```
Comment (datalogics-josepha) [inline apdfl18/Release_Notes.html:30]: What package? Nuget?

File: apdfl18/Release_Notes.html [hunk 2]
```diff
@@ -23,7 +23,26 @@ <h1>Release Notes, Adobe PDF Library v18</h1>
...
+<li>Creates new package Adobe.PDF.Library.SampleInput containing the Sample Input file.</li>
```
Comment (datalogics-josepha) [inline apdfl18/Release_Notes.html:31]: What package? Nuget?

Comment (datalogics-pgallot) [reply apdfl18/Release_Notes.html:31]: yes, Nuget packages.

Comment (datalogics-josepha) [reply apdfl18/Release_Notes.html:31]: so we need to update that then please

**Pull request merged**